cql2pgjson/src/test/java/org/folio/cql2pgjson/CompoundIndexIT.java

```diff
@@ -50,22 +54,54 @@ public void runSetup() throws Exception {
       throw new RuntimeException(e);
     }
   }
+  private List<String> cqlc(String cql) {
```

Comment:
Can you insert a blank line between all methods for readability as proposed by FOLIO style conventions?
* https://dev.folio.org/guidelines/contributing/#style-conventions
* https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141388.html#487